client/components/chat-button/index.tsx

```diff
@@ -8,10 +8,12 @@ import type { MessagingGroup } from '@automattic/help-center/src/hooks/use-messa
 import type { FC } from 'react';
 
 type ChatIntent = 'SUPPORT' | 'PRESALES' | 'PRECANCELLATION';
+export type KeyType = 'akismet' | 'jpAgency' | 'jpCheckout' | 'jpGeneral' | 'wpcom';
```

Comment:
Not sure this actually needs to be exported, perhaps I could import it from where its originally defined at `client/lib/presales-chat/index.ts`.